editor/Editor.tsx

```diff
@@ -332,12 +332,17 @@ function Editor(props: RouteComponentProps) {
             // gosRef.current.api.subscribe('trackClick', (type, eventData) => {
             //     console.warn(type, eventData.id, eventData.spec, eventData.shape);
             // });
+            // Location API
+            // gosRef.current.api.subscribe('location', (type, eventData) => {
+            //     console.warn(type, eventData.id, eventData.genomicRange);
+            // });
```

Comment:
This is where I've been testing the new location API. I noticed that there was commented out code for the other subscriptions so I did the same.